doc/Create-rosa-hcp.md

```diff
@@ -2,64 +2,82 @@
 
 ## Prerequisites
 
-1. Complete the prerequisite actions listed in [Set up to use ROSA cli](https://docs.aws.amazon.com/rosa/latest/userguide/set-up.html).
+1. Complete the prerequisite actions listed in the [Set up to use the ROSA CLI](https://docs.aws.amazon.com/rosa/latest/userguide/set-up.html) documentation.
 
-1. Create Amazon VPC that will be used with ROSA HCP using terraform template as follow;
-    1. Install the Terraform CLI. For more information, see the install instructions in the [Terraform documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli).
+2. Create an Amazon VPC that will be used with ROSA HCP using the Terraform template as follows:
+    1. Install the Terraform CLI. For more information, see the installation instructions in the [Terraform documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli).
 	1. Open a terminal session and clone the Terraform VPC repository.
 
 	   `git clone https://github.com/openshift-cs/terraform-vpc-example`
 	
-	1. Follow the commands below to create the VPC
+	1. Follow the commands below to create the VPC:
 	   ```
 	   cd terraform-vpc-example
 	   terraform init
 	   terraform plan -out rosa.tfplan -var region=<region>
 	   terraform apply rosa.tfplan
 	   ```
-	   After terraform creating the VPC you should have the subnet-ids as below
-	   ```
-       private-subnet-id: subnet-0889990000000000
-	   public-subnet-id: subnet-054ad00000000000
+       The subnet-ids will be displayed once Terraform has successfully created the VPC.	   
+       ```     
+       private-subnet-id: subnet-0889990000000000   
+       public-subnet-id: subnet-054ad00000000000
        ...
 	   ```
-1. Create the required IAM roles and OpenID Connect configuration
-   1. Create the required IAM account roles and policies.
-
-      `rosa create account-roles --force-policy-creation`
+3. Create the required IAM roles and OpenID Connect configuration.
```

Comment:
the format is not correct check it [here](https://github.com/tinaafitz/cluster-api-installer/blob/create-cluster-doc-changes/doc/Create-rosa-hcp.md)